MathLib/Integration/GaussLegendre.h

```diff
@@ -29,6 +29,15 @@ struct GaussLegendre {
     static const double W[Order];
 };
 
+template <> double const GaussLegendre<1>::X[1];
```

Comment:
This is a declaration. In the cpp we have a definition...
It is resolving the warning (this one is for `W` but same is for `X`):
```
ogs/NumLib/Fem/Integration/IntegrationGaussRegular-impl.h:82:27: warning: instantiation of variable 'MathLib::GaussLegendre<4>::W' required here, but no definition is available [-Wundefined-var-template]
        weight *= Method::W[pos[d]];
                          ^
/home/naumov/w/ogs/MathLib/Integration/GaussLegendre.h:29:25: note: forward declaration of template entity is here
    static const double W[Order];
                        ^
/home/naumov/w/ogs/NumLib/Fem/Integration/IntegrationGaussRegular-impl.h:82:27: note: add an explicit instantiation declaration to suppress this warning if 'MathLib::GaussLegendre<4>::W' is explicitly instantiated in another translation unit
        weight *= Method::W[pos[d]];
```